### PR TITLE
Update URD guides (use LSP20)

### PR DESCRIPTION
**Pull request by b00ste**
What does this PR introduce?

In this PR the URD guides are updated to use the new way of interacting with a UP that has a Key Manager as owner, which was introduced with the LSP20.

### CODE REVIEW

File: docs/guides/universal-receiver-delegate/accept-reject-assets.md [hunk 1]
```diff
@@ -349,20 +299,11 @@ const universalProfile = new ethers.Contract(
...
-
-// encode setData Calldata on the Vault
-const setDataCalldata = await universalProfile.interface.encodeFunctionData(
-  'setData(bytes32,bytes)',
-  [URD_DATA_KEY, universalProfileURDAddress],
-);
 
 // execute the executeCalldata on the Key Manager
-await keyManager.connect(EOA)['execute(bytes)'](executeCalldata);
+await keyManager
```
Comment (YamenMerhi) [inline docs/guides/universal-receiver-delegate/accept-reject-assets.md:304]: needs to be universalProfile

File: docs/guides/universal-receiver-delegate/accept-reject-assets.md [hunk 2]
```diff
@@ -174,93 +172,52 @@ const EOA = new ethers.Wallet(privateKey).connect(provider);
...
 </Tabs>
 
-### Create contract instances
+### Create UP contract instance
 
-At this point we need to create instances of the following contracts:
-
-- [**Universal Profile**](../../standards/universal-profile/lsp0-erc725account.md)
-- [**Key Manager**](../../standards/universal-profile/lsp6-key-manager.md)
+At this point we need to create instance of the [**Universal Profile**](../../standards/universal-profile/lsp0-erc725account.md) contract:
```
Comment (CJ42) [inline docs/guides/universal-receiver-delegate/accept-reject-assets.md:177]:
```suggestion
At this point we need to create an instance of the [**Universal Profile**](../../standards/universal-profile/lsp0-erc725account.md) contract:
```

File: docs/guides/universal-receiver-delegate/accept-reject-assets.md [hunk 3]
```diff
@@ -174,93 +172,52 @@ const EOA = new ethers.Wallet(privateKey).connect(provider);
...
-  [URD_DATA_KEY, universalProfileURDAddress],
-);
 ```
 
   </TabItem>
 
 </Tabs>
 
-### Execute via the Key Manager
+### Update the profile data
```
Comment (CJ42) [inline docs/guides/universal-receiver-delegate/accept-reject-assets.md:207]: We need to be specific here:

```suggestion
### Setup the LSP1 Universal Receiver Delegate
```

File: docs/guides/universal-receiver-delegate/accept-reject-assets.md [hunk 4]
```diff
@@ -124,17 +124,16 @@ npm install ethers @lukso/lsp-smart-contracts
 
 ### Imports, contants and EOA
```
Comment (CJ42) [inline docs/guides/universal-receiver-delegate/accept-reject-assets.md:125]:
```suggestion
### Imports, constants and EOA
```

**Review by CJ42 — APPROVED** [review]

**Pull request merged**